src/types/onyx/Report.ts

```diff
@@ -37,6 +37,9 @@ type Report = {
     /** The last time the report was visited */
```

Comment:
Can we please update the comment here - 
My suggestion:
`The time when user read the last message`